triage/Dockerfile

```diff
@@ -22,7 +22,7 @@ COPY . /test-infra/
 
 # Build Triage
 RUN cd /test-infra/triage/ && \
-    go build -v -o triage ./main.go
+    env GOOS=linux go build -v -o triage ./main.go
```

Comment:
Do we still need the `apk add python2` below?

Reply:
nvm!

Reply:
Yes, for the Cloud SDK